### DISCUSSION TITLE
QFlore et QFaune

### DISCUSSION THREAD

**Discussion by orovellotti** [PnX-SI/GeoNature#3328] [Ideas]
Bonjour 

J'ai rencontre @FelixHinckel qui développe ces outils avec [QFiled](https://qfield.org/):

https://github.com/FelixHinckel/QFlore
https://github.com/FelixHinckel/QFaune

Ici en version web
![image](https://github.com/user-attachments/assets/4b0fab63-e309-4b27-92c1-6be63ad340f7)

J'aimerais avoir votre avis pour leur intégration à l'écosystème en remplacement d'occTax Mobile ?

**Comment by camillemonchicourt** [comment]
Nous n'avons pas prévu de remplacer Occtax-mobile qui fonctionne bien et répond bien au besoin, et sur lequel il y a différents évolutions prévues et d'autres souhaitées.
Mais on serait preneur de plus d'informations sur les potentiels connexions/interfaçages entre ces outils QFlore et QFaune avec GeoNature.

**Comment by orovellotti** [comment]
Il me semble que Occtax-mobile ne gère que les données opportunistes et pas module monitoring nécessaire pour les BEs

**Comment by FelixHinckel** [comment]
Bonjour à tous!

QFaune et QFlore sont des projets OpenSource déjà utilisés par des faunistes et des botanistes, indépendants ou en BE. L'avantage qu'ils y trouvent par rapport à d'autres applications de relevés, c'est l'interfaçage avec QGis, et la liberté que ça amène. En plus de simple saisies opportunistes, ils peuvent renseigner des protocoles (IPA ou repasse en avifaune, transects de suivi lépidoptères, relevés phytosocios par strate en flore, ou encore de la cartographie d'habitat), et ajouter leurs données métier (zones d'étude, visualisation des zones de travaux, pré-cartographie d'habitat, observations de la bibliographies, points d'intérêt...).
L'objectif de ces projets est de proposer des outils tout en un pour les études d'impact, utilisables sur le terrain avec QField et au bureau avec QGis. 
Je commence aussi à réfléchir à l'interopérabilité avec les autres outils naturalistes. Par exemple, il existe maintenant un outil pour exporter les données recueillies au format DepoBio.

L'idée serait de réfléchir à un lien entre QFaune/QFlore et GéoNature. Par exemple, un plugin Qgis qui permet d'envoyer les données saisies dans QField directement dans une instance GéoNature. Cela permettrait aux BE de gérer la saisie naturaliste, le traitement des données, la production des cartes réglementaires, et l'export vers Géonature à partir du même outil.

**Comment by camillemonchicourt** [comment]
Occtax-mobile est le pendant mobile du module Occtax.
Il s'enrichit actuellement avec les champs additionnels dynamiques, les listes de taxons par JDD, etc...
Selon l'utilisateur authentifié, celui-ci peut se voir proposer des JDD différents et donc des listes de taxons et des champs additionnels dynamiques différents.
Il interroge automatiquement, régulièrement et directement la BDD GeoNature pour en récupérer les référentiels et la configuration. Et il alimente automatiquement une BDD GeoNature avec un fonction offline/online transparent.
Il est pensé pour un ensemble d'agents ou membres d'un collectif qui doivent avoir des outils communs et cohérents, administrés de manière globale.

Il faudrait voir ce qu'il est possible de faire sur des différents aspects avec QField, mais il me semble que l'approche et le fonctionnement est assez différent et plus individualisé et avec une synchro transparente et dynamique moins simple.

Mais au plaisir d'en discuter et d'échanger sur le sujet.

----------------------

En effet, Occtax-mobile est comme le module Occtax centré sur les données opportunistes et les inventaires, autour du standard Occurrences de taxons.

Pour les protocoles de suivi qui sont structurés autour du module Monitoring et du principe de champs additionnels dynamiques par protocole sur Sites, visites et/ou observations.
Là aussi les protocoles et leurs champs additionnels dynamiques sont administrés globalement au niveau du serveur.

Pour son équivalent mobile, on a développé une première solution fonctionnelle mais un peu complexe avec ODK : https://github.com/PnX-SI/odk2gn

RNF a le projet de faire une application Monitoring-mobile qui a été évoquées lors des rencontres GeoNature 2024 (voir "16h30 : Retour d'expérience du réseau des réserves naturelles / Projet Monitoring-mobile (Zacharie Moulin - RNF)" sur https://geonature.fr/rencontres-2024.html.

**Comment by vajapavlovic** [comment]
Effectivement, il est souvent remonté par les bureaux d'études (BE) le besoin d'avoir une application utilisable hors ligne qui offrirait les mêmes fonctionnalités que le module Monitoring. Cela permettrait de saisir des données protocolées directement sur le terrain, sans nécessiter une connexion Internet.

De plus, il pourrait être pertinent de permettre aux naturalistes de conserver les habitudes qu'ils ont déjà développées avec des outils comme QFlore et QFaune, tout en leur donnant la possibilité d'utiliser GeoNature. Cela pourrait être réalisé en intégrant une option de connexion entre ces outils et GeoNature.

Cette approche renforcerait l'adoption de GeoNature en s'adaptant aux pratiques déjà bien établies des utilisateurs tout en facilitant la transition.

**Comment by camillemonchicourt** [comment]
Oui c'est intéressant.
Je pense que c'est une solution, un fonctionnement, une chaîne de travail et des objectifs bien différents de Occtax-mobile ou même Monitoring, mais potentiellement complémentaire.
On pourrait en discuter lors d'un COTECH si vous le souhaitez.
Comme rappelé lors des rencontres GeoNature, toute structure peut proposer un comité technique sur le sujet qu'il souhaite (https://geonature.fr/documents/rencontres/2024/ateliers/Atelier-Gouvernance.pdf). 

Le créneau habituel est le mardi de 13h30 à 14h (on déborde avec ceux qui veulent/peuvent si besoin) en visio et viennent ce qui veulent/peuvent.

On peut caler la date ici ou alors directement sur le salon Matrix dédié : https://matrix.to/#/#geonature-dev:matrix.org

Je vous laisse donc proposer un créneau si vous le souhaitez.

**Comment by MoulinZ** [comment]
Effectivement nous avons, à RNF, engagé le développement d'une application mobile pour le module monitorings (développé par @aschlegle ). Une première version bêta de l'outil devrait sortir début 2025.
